MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXResultViewModel.swift

```diff
@@ -29,6 +29,13 @@ internal class PXResultViewModel: NSObject {
         self.remedy = remedy
         self.oneTapDto = oneTapDto
     }
+    
+    func selectPaymentMethodConsumerCredits() -> Bool {
```

Comment:
Acho que fica mais enxuto assim, repare que a condição dentro do if já é a resposta que queremos ;)
```suggestion
    var isConsumerCredits: Bool {
     paymentResult.paymentMethodId == PXPaymentTypes.CONSUMER_CREDITS.rawValue
    }
```

Reply:
Faz todo sentido....estou vendo isso agora.

Reply:
Na verdade, vou excluir o metodo, pois não vou mais utilizar, pois a regra que dependia dessa validação foi excluida no refactor de ontem...

Reply:
feito